src/App.tsx

```diff
@@ -27,8 +28,12 @@ const App = withProviders(() => {
   return (
     <ThemeProvider theme={theme}>
       <div style={appStyle} data-net={isMainnet() ? 'mainnet' : 'testnet'}>
-        <Routers />
-        <Toast />
+        {initHardForkStatus && (
```

Comment:
Is `initHardForkStatus` used to check if `statistics` has been updated and left time has been calculated? Maybe we can cache the `isHardForkFinished`, or the page will be blank if this API is unreachable.

Reply:
yeah, it's a good idea to cache hard fork info.Then users maybe only see incorrect page first time